### ISSUE TITLE
Tipbot lottery

### ISSUE THREAD

**Issue by acetmesis** [noescape00/DiscordTipBot#33]
Suggestion from Bogdanel from the discord, a weekly tipbot lottery. Participants can send 1 Strat once to a lottery during the week. At the end of the week winners are chosen at random from amongst the participants and they receive a share of the Strat. Say, 1st place 60%, 2nd place 30%, 3rd place 10%

**Comment by noescape00** [comment]
Yeah, I don't mind having it. But I have a feeling that not many ppl will participate. 

Also imho winning chance should depend on % of total sum of money to be distributed.
